tests/external/test_scar.py

```diff
@@ -0,0 +1,19 @@
+from scvi.data import synthetic_iid
+from scvi.external import SCAR
+
+
+def test_scar():
+    n_latent = 5
+    adata = synthetic_iid()
+    SCAR.setup_anndata(adata, batch_key="batch", labels_key="labels")
+
+    model = SCAR(adata, ambient_profile=None, n_latent=n_latent)
+    model.train(1, check_val_every_n_epoch=1, train_size=0.5)
+    model.get_elbo()
+    model.get_latent_representation()
+    model.get_marginal_ll(n_mc_samples=5)
+    model.get_reconstruction_error()
+    model.history
```

Comment:
how do you get the denoised counts out of the model?

Reply:
added as a test in latest commit with an example in the user guide